raiden/network/rpc/client.py

```diff
@@ -621,6 +621,9 @@ def new_netting_channel(self, peer1, peer2, settle_timeout):
         else:
             other = peer1
 
+        if self.proxy.getChannelWith(other) != '0' * 40:
```

Comment:
It's not a bad idea to solve it the way that you did, but with the way that our current implementation works, it just won't work. :(